src/main/java/spoon/reflect/factory/Factory.java

```diff
@@ -156,6 +157,8 @@
 
 	QueryFactory Query();
 
+	FineModelChangeListener Change();
```

Comment:
remove this change? The entry point is now clearly in Environment. (Thanks revapi)